Apps/WebClient/src/Server/Controllers/RatingController.cs

```diff
@@ -0,0 +1,68 @@
+﻿// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+namespace HealthGateway.WebClient.Controllers
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Security.Claims;
+    using System.Threading.Tasks;
+    using HealthGateway.Common.AccessManagement.Authorization;
+    using HealthGateway.Common.Filters;
+    using HealthGateway.Common.Models;
+    using HealthGateway.Database.Models;
+    using HealthGateway.WebClient.Models;
+    using HealthGateway.WebClient.Services;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+
+    /// <summary>
+    /// Web API to handle user ratings.
+    /// </summary>
+    [Authorize]
+    [ApiVersion("1.0")]
+    [Route("v{version:apiVersion}/api/[controller]")]
+    [ApiController]
+    public class RatingController
+    {
+        private readonly object ratingService; // TODO: Update with Interface.
```

Comment:
re. the development process, does this mean that we need to get another PR when the service is completed right?

Reply:
I've added the service interface!